guides-markdown/working-with-txs.md

```diff
@@ -0,0 +1,173 @@
+# Working with transactions
+```post-author
+Christopher Jeffrey
+```
+```post-description
+Working with transaction with bcoin, MTX and TX objects.
```

Comment:
Change to "Working with transactions using bcoin and the MTX and TX objects."

Reply:
ok